### ISSUE TITLE
Batch images update command

### ISSUE THREAD

**Issue by beeing** [pdfcpu/pdfcpu#1035]
Related to #455 in which the `pdfcpu images extract` will extract all images from PDF file to a target directory.

However, I noticed that there's no batch command to perform images update from the directory that was extracted to.

Will be nice to have the `pdfcpu images update source.pdf {imagesDir}` which based on the extracted filenames to update the images back to the PDF.

This will be handy for external images processing (such as compression / watermark) and then update back the images to replace the one in the PDF file.

The batch function should read files from PNG or JPG extensions and parse the page / image and Id and use that info to update the images. Unparseable filenames will be ignored and shown in the stdout.

Thanks in advance.

**Comment by hhrutter** [comment]
This is a nice to have but has to wait until we get more expected feedback related to updating images.

**Comment by bckmnn** [comment]
I also think this would be a nice feature. However it is also quite trivial to do it like this: 

```sh
mkdir extracted_images

# extract images
pdfcpu extract -mode image input.pdf extracted_images

# edit images 
#  > change compression / format  / add watermark /... 
# > dimension needs to be the same as before 
mogrify -path extracted_images -quality 75 extracted_images/*.jpg

# import again into pdf
# > be aware that this updates input.pdf
for img in extracted_images; do
    pdfcpu images update input.pdf "$img"
done
```
thats what i am currently using for my usecase which is mostly recompression.

**Comment by hhrutter** [comment]
I agree with @bckmnn this is smth anybody can build on top of pdfcpu easily.
